tests/cql/CqlListOperatorsTest.xml

```diff
@@ -565,7 +585,7 @@
 		</test>
 		<test name="EquivalentDateTimeNull">
 			<expression>{DateTime(2001, 9, 11), DateTime(2012, 5, 10), DateTime(2014, 12, 10)} ~ {DateTime(2001, 9, 11), DateTime(2012, 5, 10), DateTime(2014, 12, 10), null}</expression>
-			<output>null</output>
+			<output>false</output>
```

Comment:
According to the spec:
> The equivalent (~) operator for lists returns true if and only if the lists contain elements of the same type, have the same number of elements, and for each element in the lists, in order, the elements are equivalent.

The first list has 3 items and the second list has 4 items, so they are definitely not equivalent.